pkg/controller/defaults/resources.go

```diff
@@ -79,10 +79,20 @@ var (
 				corev1.ResourceMemory: resource.MustParse("4Gi"),
 			},
 		},
-		"noobaa-db-vol": corev1.ResourceRequirements{
+		"noobaa-db-vol": {
 			Requests: corev1.ResourceList{
 				corev1.ResourceStorage: resource.MustParse("50Gi"),
 			},
 		},
+		"noobaa-endpoint": {
```

Comment:
@nb-ohad should we re-adjust the noobaa-core resources now that endpoints have split from it?

Reply:
@guymguym I was thinking this is the readjusted resources, Please correct me If I am wrong.

Reply:
Better check it